blog-service/2023-06-07-alerts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Muting Schedules (Alerts)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - monitors
+  - alerts
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+New - we're happy to announce the release of **Muting Schedules**, a new feature that allows you to pause monitor notifications according to a schedule that you define. You can apply this setting to individual monitors and/or folders. For folders, this will mute all nested monitors and subfolders.
```

Comment:
```suggestion
We're happy to announce the release of **Muting Schedules**, a new feature that allows you to pause monitor notifications according to a schedule that you define. You can apply this setting to individual monitors and/or folders. For folders, this will mute all nested monitors and subfolders.
```

Reply:
@jpipkin1 our Service release notes style is to write "New -" or "Updated -" at the beginning. It looks like we've got away from it this year, but you can see it in [previous years](https://help.sumologic.com/release-notes-service/2022/12/31/). I'll add this to the agenda.